test/unittests/NDB_BVL_Instrument_LINST_ToJSON_Test.php

```diff
@@ -60,7 +60,12 @@ function _getAllMethodsExcept($methods) {
     function testMetaData() {
         $instrument = "table{@}Test\ntitle{@}Test Instrument";
         $base64 = "data://text/plain;base64," . base64_encode($instrument);
-        $this->i->loadInstrumentFile($base64, true);
+        try {
```

Comment:
since these try-catch blocks are not important for the test. why don't you remove it?
it always catches the exception and makes loadInstrumentFile function failed.